webapp-django/crashstats/crashstats/templates/crashstats/crashes_per_user.html

```diff
@@ -0,0 +1,275 @@
+{% extends "crashstats_base.html" %}
+{% block site_css %}
+  {{ super() }}
+  {% compress css %}
+      <link rel="stylesheet" type="text/css" href="{{ static('crashstats/css/lib/jquery-ui.css') }}" media="screen" />
+      <link rel="stylesheet" type="text/css" href="{{ static('crashstats/css/lib/jquery-ui.structure.css') }}" media="screen" />
+      <link rel="stylesheet" type="text/css" href="{{ static('crashstats/css/lib/jquery-ui.theme.css') }}" media="screen" />
+      <link rel="stylesheet" type="text/css" href="{{ static('crashstats/css/lib/metricsgraphics.css') }}">
+      <link rel="stylesheet" type="text/css" href="{{ static('crashstats/css/metricsgraphics_custom.css') }}">
+  {% endcompress %}
+  {% compress css %}
+      <link rel="stylesheet" type="text/less" href="{{ static('crashstats/css/crashes_per_user.less') }}">
+  {% endcompress %}
+{% endblock %}
+
+{% block page_title %}
+Crashes per User for {{ product }}
+{% endblock %}
+
+{% block content %}
+<div id="mainbody">
+
+<div class="page-heading">
+    <p style="float: right">
```

Comment:
I'm pretty sure that would feel a lot better in the ``less`` file. :) 

(Unless you are concerned about us forgetting to remove the associated rule when we remove this link. Which sounds valid to me... This shouldn't stick around for too long so I suppose it's fine. )